worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/DistributorModule.java

```diff
@@ -156,13 +162,26 @@ ConnectionFactory provideConnectionFactory(final MessageDistributorConfig messag
         ConnectionFactory connectionFactory = new ConnectionFactory();
 
         try {
+            if (messageDistributorConfig.getRabbitmqProtocol().equalsIgnoreCase("amqps")) {
+                final String tmfAlgorithm = System.getProperty(
+                        TRUST_MANAGER_FACTORY_ALGORITHM, TrustManagerFactory.getDefaultAlgorithm()
```

Comment:
While I was reviewing the release notes I wanted to see what the "default" was. 
I looked at the implementation of TrustManagerFactory.getDefaultAlgorithm() and I see there's already a way to externally configure a value for `ssl.TrustManagerFactory.algorithm` (See Security.java initialize())

Let's remove this additional way.